src/utils/entraApp.ts

```diff
@@ -410,5 +412,28 @@ export const entraApp = {
     }
 
     return resolvedApis;
+  },
+  async getAppObjectIdFromAppId(appId: string): Promise<string> {
+    const apps = await odata.getAllItems<Application>(`https://graph.microsoft.com/v1.0/applications?$filter=appId eq '${appId}'&$select=id`);
+
+    if (apps.length === 0) {
+      throw `App with appId '${appId}' not found in Microsoft Entra ID`;
+    }
+
+    return apps[0].id!;
+  },
+  async getAppObjectIdFromAppName(appName: string): Promise<string> {
```

Comment:
Let's change the interface of this helper function a bit to make it more broadly applyable:


```suggestion
  async getAppRegistrationByName(displayName: string, properties?: string[]): Promise<string> {
```

Let's return the entire object and let's make it possible to $select specific properties.